repository-data/webfiles/src/main/resources/site/freemarker/common/static-ui/shared/nav.ftl

```diff
@@ -3,34 +3,34 @@
 <nav class="static-ui-menu">
     <ul class="list list--reset">
         <li>
-            <a href="/static-ui/home">Home page</a>
+            <a href="/site/static-ui/home">Home page</a>
```

Comment:
These links will not work on the TST, DEV, etc. servers. You need to use somthing like:

```text
<a href="<@hst.link path='/static-ui/home'/>">Home page</a>
```

Then let the CMS resolve the path for the environment's config.

Reply:
This is done, and it works for the root `static ui` links. However when I do the same update to more nested routes such as `static-ui/component/block-link` in the `component/index.ftl` file, the links resolve as `/static-ui/component`.

Reply:
Will they ever be put onto 'tst' or 'dev' for the P.O. to view it?

Reply:
We don't need it, this is only for local development. If there is a quick way of getting this working 'the right' way, let's do it, but I'll need your help with it, as my attempt didn't work :)